sdk/resourcemanager/Azure.ResourceManager/src/ArmOperation.cs

```diff
@@ -3,13 +3,34 @@
 
 // <auto-generated/>
 
-#nullable disable
+#nullable enable
```

Comment:
I don't think we want to turn this on as its a breaking change from my understanding flipping to nullable enabled.

Reply:
Don't think this is a breaking change, but yeah we can keep it as it is.

Reply:
reverted